onnx/defs/math/defs.cc

```diff
@@ -644,6 +644,33 @@ ONNX_OPERATOR_SET_SCHEMA(
         .SetContextDependentFunctionBodyBuilder(BuildContextDependentFunctionBodyGelu)
         .TypeAndShapeInferenceFunction(propagateShapeAndTypeFromFirstInput));
 
+static const char* Swish_ver21_doc = R"DOC(
+Swish function takes one input data (Tensor<T>) and produces one output data (Tensor<T>) of the same shape,
+where $Swish(x) = x * sigmoid(beta * x)$.
+)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+    Swish,
+    21,
+    OpSchema()
+        .Attr("beta", "Value of beta.", AttributeProto::FLOAT, 1.0f)
+        .SetDoc(Swish_ver21_doc)
+        .Input(0, "X", "Input tensor", "T", OpSchema::Single, true, 1, OpSchema::Differentiable)
+        .Output(0, "Y", "Output tensor", "T", OpSchema::Single, true, 1, OpSchema::Differentiable)
+        .TypeConstraint(
+          "T",
+            {"tensor(float16)", "tensor(float)", "tensor(bfloat16)"},
+            "Constrain input and output types to float tensors.")
+        .TypeAndShapeInferenceFunction(propagateShapeAndTypeFromFirstInput)
+        .FunctionBody(
+            R"ONNX(
+          {
+            S_X = Sigmoid<beta = 1.0>(X)
```

Comment:
Sigmoid doesn't have an attribute.  We should be doing something like below:
```py
   Beta = Constant <value_float: float = @beta>()
   BetaCast = CastLike (Beta, X)
   BetaX = Mul (BetaCast, X)
   SigmoidBetaX = Sigmoid(BetaX)
   Y = Mul (X, SigmoidBetaX)
```

Reply:
However, as commented above, it seems to me we should be calling this `alpha` instead of `beta`, based on the convention in `HardSwish`.